android/app-example/src/main/java/com/badoo/ribs/example/rib/dialog_example/DialogExampleRouter.kt

```diff
@@ -7,27 +7,31 @@ import com.badoo.ribs.core.routing.action.RoutingAction
 import com.badoo.ribs.core.routing.action.RoutingAction.Companion.noop
 import com.badoo.ribs.dialog.DialogLauncher
 import com.badoo.ribs.example.rib.dialog_example.DialogExampleRouter.Configuration
+import com.badoo.ribs.example.rib.dialog_example.dialog.LazyDialog
 import com.badoo.ribs.example.rib.dialog_example.dialog.SimpleDialog
 import com.badoo.ribs.example.rib.dialog_example.dialog.RibDialog
 import kotlinx.android.parcel.Parcelize
 
 class DialogExampleRouter(
     private val dialogLauncher: DialogLauncher,
     private val simpleDialog: SimpleDialog,
+    private val lazyDialog: LazyDialog,
     private val ribDialog: RibDialog
 ): Router<Configuration, DialogExampleView>(
     initialConfiguration = Configuration.Default
 ) {
     sealed class Configuration : Parcelable {
         @Parcelize object Default : Configuration()
         @Parcelize object SimpleDialog : Configuration()
+        @Parcelize object LazyDialog : Configuration()
         @Parcelize object RibDialog : Configuration()
     }
 
     override fun resolveConfiguration(configuration: Configuration): RoutingAction<DialogExampleView> =
         when (configuration) {
-            Configuration.Default -> noop()
-            Configuration.SimpleDialog -> showDialog(dialogLauncher, simpleDialog)
-            Configuration.RibDialog -> showDialog(dialogLauncher, ribDialog)
+            is Configuration.Default -> noop()
+            is Configuration.SimpleDialog -> showDialog(dialogLauncher, simpleDialog)
+            is Configuration.LazyDialog -> showDialog(dialogLauncher, lazyDialog)
+            is Configuration.RibDialog -> showDialog(dialogLauncher, ribDialog)
```

Comment:
Why these configurations are not overlays?

Reply:
Because I forgot! Good point, will fix.

Reply:
Fixed.